GOVERNANCE.md

```diff
@@ -0,0 +1,122 @@
+  * Project: A single repository in the [Fn Project GitHub organization](https://github.com/fnproject).
+  * The Fn Project: The sum of all activities performed under this governance, concerning one or more repositories or the community. 
+
+## Values
+Developers and community are expected to follow the values defined in the [Fn Project Code Of Conduct](CODE_OF_CONDUCT.md). Furthermore, the Fn Project community strives for kindness, giving feedback effectively, and building a welcoming environment. The Fn Project developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Projects
+Each project must have a `MAINTAINERS.md` file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Major releases should be announced in the [Release category in forums.fnproject.io](https://forums.fnproject.io/c/release) . Any new projects should be first proposed in the [Fn Project forums](https://forums.fnproject.io/) following the voting procedures listed below. When a project is no longer relevant it should be moved to the fnproject-junkyard GitHub organization.
+
+## Team members
+Team member status may be given to those who have made ongoing contributions to the Fn Project for at least 3 months. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support will also be taken into account.
+
+New members may be proposed by any existing member by posting to the Fn Project forums. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
+
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or decline their acceptance of team membership. This will also be posted to the Fn Project forums for record-keeping purposes.
+
+If they choose to accept, the following steps are taken:
+  * Team members are added to the [GitHub organization](https://github.com/fnproject) as _Maintainer_.
+  * Team members are added to the [Fn Team group on forums.fnproject.io](https://forums.fnproject.io/groups/FnTeam).
+  * New team members are announced on the Fn Project forums by an existing team member.
+
+Team members may retire at any time by posting to the Fn Project forums and removing themselves from the [Fn Team group](https://forums.fnproject.io/groups/FnTeam).
+
+Team members can be removed by [supermajority vote](#supermajority-vote) in the [private Fn Team category in the Fn Project forums](https://forums.fnproject.io/c/fn-team). For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
```

Comment:
s/their team membership ends automatically/You will ride eternal, shiny and chrome!/ :)